samples/snippets/src/test/java/pubsub/AdminIT.java

```diff
@@ -208,17 +249,30 @@ public void testAdmin() throws Exception {
         .contains("Created a subscription with exactly once delivery enabled:");
     assertThat(bout.toString()).contains("enable_exactly_once_delivery=true");
 
+    bout.reset();
+    // Test create a BigQuery subscription
+    createBigQueryTable();
+    String bigqueryTablePath = String.join(".", projectId, bigqueryDatasetId, bigqueryTableId);
+    CreateBigQuerySubscriptionExample.createBigQuerySubscription(
+        projectId, topicId, bigquerySubscriptionId, bigqueryTablePath);
+    assertThat(bout.toString()).contains("Created a BigQuery subscription:");
+    assertThat(bout.toString()).contains(bigqueryTablePath);
+
     bout.reset();
     // Test delete subscription.
     DeleteSubscriptionExample.deleteSubscriptionExample(projectId, pullSubscriptionId);
     DeleteSubscriptionExample.deleteSubscriptionExample(projectId, pushSubscriptionId);
     DeleteSubscriptionExample.deleteSubscriptionExample(projectId, orderedSubscriptionId);
     DeleteSubscriptionExample.deleteSubscriptionExample(projectId, exactlyOnceSubscriptionId);
+    DeleteSubscriptionExample.deleteSubscriptionExample(projectId, bigquerySubscriptionId);
     assertThat(bout.toString()).contains("Deleted subscription.");
 
     bout.reset();
     // Test delete topic.
     DeleteTopicExample.deleteTopicExample(projectId, topicId);
     assertThat(bout.toString()).contains("Deleted topic.");
+
+    // Delete BigQuery table.
+    deleteBigQueryTable();
```

Comment:
Move this to the cleanup step: https://github.com/googleapis/java-pubsub/blob/df8a01da0ac2f5120a745cc5013a9603cf0ccdb1/samples/snippets/src/test/java/pubsub/AdminIT.java#L99-L100